terraform/environments/ops/main.tf

```diff
@@ -7,12 +7,18 @@ module "emblem_ops" {
   repo_name       = var.repo_name
 }
 
+data "google_cloud_run_service" "website_e2e_test_content_api" {
```

Comment:
rename to what it is not what it'll be used for eg `google_cloud_run_service.content_api`

Reply:
Done. (I added a comment indicating that this was for Website E2E testing instead.)